handlers/line_item_handler.py

```diff
@@ -1,80 +1,33 @@
-
-#     ########## BUILD POPULARITY DICT ##########
-#     # create dictionary with keys: product ids and values: dict of purchase info
-#     li_dict = {obj.product_id: {"qty": 0, "customers": set(), "revenue": 0} for uid, obj in li_lib.items()}
-#     # loop through all line items and populate corresponding product keys with appropriate info
-#     for uid, obj in li_lib.items():
-#         customer = orders_lib[obj.order_id].customer_id
-#         li_dict[obj.product_id]["qty"] += 1
-#         li_dict[obj.product_id]["customers"].append(customer)
-#     # calculate revenue
-#     for product, info in li_dict.items():
-#         price = products_lib[product].price
-#         info["revenue"] = info["qty"] * price
 
-#     row_string = "{0:<18}{1:<11}{2:<11}${3:<14}"
-#     total_string = "{0:<18}{1:<11}{2:<11}${3:<14}"
+    sqlite3.connect(input_file) as conn:
+        db = conn.cursor()
+        db.execute("INSERT INTO LineItem (OrderId, ProductId) VALUES (order_id, product_id)")
+        db.commit()
 
-#     ########## PRINT REPORT ##########
-#     print(total_string.format("Products", "Orders", "Customers", "Revenue"))
-#     print("*" * 55)
-#     order_list, customer_list, revenue_list = [],[],[]
-#     for product, info in li_dict.items():
-#         # set product names
-#         product_name = products_lib[product]['name']
-#         order = info["qty"]
-#         customers = info["customers"]
-#         revenue = info["revenue"]
+    return
 
-#         # add values to lists (to be used in totals calculation)
-#         order_list.append(order)
-#         customer_list.append(customer)
-#         revenue_list.append(revenue)
 
-#         # limit display names/values
-#         product_name = (product_name if len(product_name) <= 17 else product_name[:14] + "...") + " "
-#         order = (order if len(order) <= 11 else order[:8] + "...") + " "
-#         customers = (customers if len(customers) <= 11 else customers[:8] + "...")
-#         revenue = (revenue if len(revenue) <= 14 else revenue[:11] + "...")
-
-#         # print product info
-#         print(row_string.format(product_name, order, customers, revenue))
-#     print("*" * 55)
-
-#     # calculate totals
-#     order_sum = sum(order_list)
-#     customer_sum = sum(customer_list)
-#     revenue_sum = sum(revenue_list)
+def return_report_line_items(input_file='bangazon.db'):
+    """
+    Returns a list of line items with orders that have been completed.
+    """
 
-#     # limit totals display
-#     order_sum = (order_sum if len(str(order_sum)) <= 17 else order_sum[:14] + "...") + " "
-#     customer_sum = (customer_sum if len(str(customer_sum)) <= 11 else customer_sum[:8] + "...")
-#     revenue_sum = (revenue_sum if len(str(revenue_sum)) <= 14 else revenue_sum[:11] + "...")
+    sqlite3.connet(input_file) as conn:
+        db = conn.cursor()
+        db.execute("SELECT li.* FROM LineItem li, Orders o WHERE li.OrderID == o.OrderID AND o.PaymentID == NULL")
```

Comment:
don't we want the ones where the payment id isn't null? if it's null, it's a cart, if it's a paymentID, then the order is completed... also is there a difference in sql between one equals sign and two?